extensions/mssql/src/hdfs/ui/uiConstants.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export namespace cssStyles {
+	export const tableBorder = '1px solid #ccc';
```

Comment:
does this color work well for both light and dark themes?

Reply:
All UI elements will be worked on in later PRs - this one is primarily focused on functionality.